image_definitions/base/Dockerfile

```diff
@@ -52,20 +52,20 @@ ENV TZ="Europe/London"
 RUN apk add --update --no-cache ca-certificates \
     && apk add -t deps \
     && apk add --no-cache --update tzdata \
-        bash \
-        curl \
-        unzip \
-        git \
-        tar \
-        python3 \
-        python3-dev \
-        py3-pip \
-        build-base \
-        linux-headers \
-        docker-cli \
-        sudo \
-        shadow \
-        gnupg \
+    bash \
+    curl \
+    unzip \
+    git \
+    tar \
+    python3 \
+    python3-dev \
+    py3-pip \
+    build-base \
+    linux-headers \
+    docker-cli \
+    sudo \
+    shadow \
+    gnupg \
```

Comment:
Wait one sec, why has this been brought back?

Reply:
Can we put the indentation back please? 😁

Reply:
yeah it auto formatting by vs code, let me try to put it back :)

Reply:
No worries, I wonder why it tries to do that =S

Reply:
done